misc/word_palindrome.c

```diff
@@ -0,0 +1,78 @@
+/**
+ * @file
+ * @brief Check if a given word or phrase is a palindrome.
+ * @details
+ * A palindrome is a word, number, phrase, or other sequence of 
+ * characters which reads the same backward as forward, such as
+ * madam, racecar.
+ * This program will be able to check phrases, with words 
+ * separated by spaces, but can't deal with special characters.
+ * @author [Maria Eduarda Kawakami](https://github.com/madukm)
+ *
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <assert.h>
+#include <string.h>
+#include <ctype.h>
```

Comment:
Add a brief description of what the headers/libraries add (see the example below).
```suggestion
#include <stdio.h>      ///
#include <stdlib.h>    ///
#include <assert.h>   /// for assert
#include <string.h>   ///
#include <ctype.h>   ///
```

Reply:
Done. Thanks :)